tests/apps/pubsub-subscriber/app.go

```diff
@@ -33,18 +33,25 @@ import (
 )
 
 const (
-	appPort          = 3000
-	pubsubA          = "pubsub-a-topic-http"
-	pubsubB          = "pubsub-b-topic-http"
-	pubsubC          = "pubsub-c-topic-http"
-	pubsubJob        = "pubsub-job-topic-http"
-	pubsubRaw        = "pubsub-raw-topic-http"
-	pubsubDead       = "pubsub-dead-topic-http"
-	pubsubDeadLetter = "pubsub-deadletter-topic-http"
-	PubSubEnvVar     = "DAPR_TEST_PUBSUB_NAME"
+	appPort            = 3000
+	pubsubA            = "pubsub-a-topic-http"
+	pubsubB            = "pubsub-b-topic-http"
+	pubsubC            = "pubsub-c-topic-http"
+	pubsubJob          = "pubsub-job-topic-http"
+	pubsubRaw          = "pubsub-raw-topic-http"
+	pubsubDead         = "pubsub-dead-topic-http"
+	pubsubDeadLetter   = "pubsub-deadletter-topic-http"
+	pubsubBulkTopic    = "pubsub-bulk-topic-http"
+	pubsubRawBulkTopic = "pubsub-raw-bulk-topic-http"
+	pubsubCEBulkTopic  = "pubsub-ce-bulk-topic-http"
+	pubsubDefBulkTopic = "pubsub-def-bulk-topic-http"
```

Comment:
Reviewers : these are the topic where bulk publish publishes to and normal subscribe subscribe to